src/main/java/com/cburch/logisim/std/ttl/Ttl7436.java

```diff
@@ -18,7 +25,46 @@ public class Ttl7436 extends Ttl7402 {
    */
   public static final String _ID = "7436";
 
+  private static class NorGateHdlGeneratorFactory extends AbstractGateHdlGenerator {
+    @Override
+    public LineBuffer getLogicFunction(int index) {
+      return LineBuffer.getHdlBuffer()
+          .add("{{assign}}gateO{{1}}{{=}}{{not}}(gateA{{1}}{{or}}gateB{{1}});", index);
```

Comment:
why not ```{{=}} gateA{{1}} {{nor}} gateB{{1}}``` ?

Reply:
this is just a copy of the code in Ttl7402, so maybe cleaning that up should be a separate issue